README.md

```diff
@@ -1885,6 +1931,8 @@ _Related: [Software Development - Project Management](#software-development---pr
 
 **[`^        back to top        ^`](#)**
 
+[Help desk](https://en.wikipedia.org/wiki/Help_desk_software) and [issue tracking](https://en.wikipedia.org/wiki/Issue_tracking_system) software to keep track of user requests and deal with other customer-care-related issues. 
```

Comment:
As this section mixes support/helpdesk-oriented ticketing systems, and bug/issue trackers, I think the description could be improved (track user requests, bugs, missing features...). Right now I can't think of a correct way to phrase it.

Reply:
`[Ticketing](https://en.wikipedia.org/wiki/Issue_tracking_system) and [Bug Tracking](https://en.wikipedia.org/wiki/Bug_tracking_system) software to help the tracking of user requests, bugs and missing features`

What about the above?

Reply:
```
[Helpdesk](https://en.wikipedia.org/wiki/Help_desk_software), [bug](https://en.wikipedia.org/wiki/Bug_tracking_system) and [issue](https://en.wikipedia.org/wiki/Issue_tracking_system) tracking software to help the tracking of user requests, bugs and missing features.
```

^ ?

Reply:
Yeah thats more fluent than what I had 😆, will add now and resolve